doc/source/whatsnew/v0.11.0.rst

```diff
@@ -305,6 +305,7 @@ Furthermore ``datetime64[ns]`` columns are created by default, when passed datet
 Astype conversion on ``datetime64[ns]`` to ``object``, implicitly converts ``NaT`` to ``np.nan``
 
 .. ipython:: python
+   :okexcept:
 
    s = pd.Series([datetime.datetime(2001, 1, 2, 0, 0) for i in range(3)])
```

Comment:
```suggestion

   import datetime
   s = pd.Series([datetime.datetime(2001, 1, 2, 0, 0) for i in range(3)])
```

Since the error was about datetime not being defined, this might also solve it?

Reply:
sure lets try!